### DISCUSSION TITLE
Terraform run is failing with token issue

### DISCUSSION THREAD

**Discussion by soundd2** [community/community#81022] [Actions]
### Select Topic Area

Show & Tell

### Body

After I merge it is supposed to run terraform but it is failing with following error. Please suggest. 

![image](https://github.com/community/community/assets/94822959/fc0f70c8-fc6f-4c1d-8bef-0b3777dd97c0)

Which token I should grab and where I should place to fix this error.

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @soundd2 ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. I've gone ahead and moved it for you. Good luck!
